en/snap/platform/overview.md

```diff
@@ -1,11 +1,11 @@
 # Integrate Midtrans Snap to 3rd Party Ecommerce Platform
 <hr>
 
-If you are using 3rd party E-commerce platform or SaaS like Shopify, Sirclo, and Jejualan. 
+Midtrans Snap can be integrated with third party E-commerce platform or SaaS like Shopify, Sirclo, and Jejualan. 
 
-We have partnered with various platfroms to make integration process as easy as possible. This page contains a list of platforms that have partnered with Midtrans, if you are using 3rd party platform that are not listed, and would like for us to integrate with it, please let us know via [support@midtrans.com](mailto:support@midtrans.com "email support")
+Midtrans is associated with various platforms to make integration process as easy as possible. This page contains a list of platforms that have partnered with Midtrans. If you are using third party platforms that are not listed here, and would like for Midtrans  to integrate with it, please contact us at [support@midtrans.com](mailto:support@midtrans.com "email support")
```

Comment:
Can we avoid the ambiguity of word `associated`, in terms of company-relationship, "associated company" could means the company have parent-child relationship. Which is not true in this case. `partnership/partner`  is more appropriate here.